editor/src/components/canvas/canvas-strategies/strategies/grid-rearrange-move-strategy.spec.browser2.tsx

```diff
@@ -506,8 +506,8 @@ export var storyboard = (
         {
           const { top, left, gridColumn, gridRow } = child.style
           expect({ top, left, gridColumn, gridRow }).toEqual({
-            gridColumn: '1',
-            gridRow: '1',
+            gridColumn: '1 / 3',
+            gridRow: '1 / 3',
```

Comment:
where do these `/ 3`s come from?

Reply:
I looked into this a bit and the element being dragged around spans over two cells in each direction.

Previously these `1`s would've been treated as `1 / 2`, but because of the change I made to `parseGridRange` it would be treated as `1 / auto`. So I assume the logic is correcting those to what they are as a result. I tried the code and moved it to other locations and these changed to `2 / 4` and so forth, I didn't dig into it any further because it appears to be correct.

Reply:
(we chatted in DMs and realized it's not based on the size of the element, the size 2 seems to come from a mistake)